### DISCUSSION TITLE
Refinement of survey responses - bug triage, issue management

### DISCUSSION THREAD

**Discussion by bertfrees** [daisy/pipeline#707] [General]
We are currently in the process of analysing your responses to the [Pipeline usage survey](https://github.com/daisy/pipeline/discussions/682) held earlier this year. Some of the feedback requires some more elaboration or discussion. Below are some responses in the topic "bug triage":

---
- **Q**: How happy do you think your organization is with Pipeline right now?
  - One organisation says it takes too long time to handle bugs, and it is too hard to get in new functionality.
---

I would like to know:

- We would like suggestions for a system to handle tickets faster (with the resources available). E.g. could we help contributors fix bugs themselves? Or do you think should we   allocate more time for handling bugs?

**Comment by egli** [comment]
IMHO people should fix the bugs themselves or pay someone to have them fixed for them
